spec.bs

```diff
@@ -616,9 +623,8 @@ To <dfn>construct a pending fenced frame config</dfn> given an [=auction config=
   : [=fenced frame config/effective enabled permissions=]
   :: a [=struct=] with the following [=struct/items=]:
     : [=effective enabled permissions/value=]
-    :: «"<code>{{PermissionPolicy/attribution-reporting}}</code>",
-        "<code>private-aggregation</code>" (TODO:ref), "<code>shared-storage</code>" (TODO:ref),
-        "<code>shared-storage-select-url</code>" (TODO:ref)»
+    :: «"<code>{{PermissionPolicy/attribution-reporting}}</code>", "[=private-aggregation=]",

```

Comment:
We might as well be consistent with our usage of `<code>` here, I think. And at least some of these definitions use `<code>` while this PR currently does not. (But even for the ones that don't, I think we should just for consistency).

Reply:
Done.

(Cannot remember what I tried but got some errors. This time it works)